File: labsC#/lab02/StringArray.cs
using System;

namespace StringLibrary
{
    public class StringArray
    {
        private string data;  

        
        public StringArray()
        {
            data = string.Empty;
        }

        
        public StringArray(string str)
        {
            data = str;
        }

        
        ~StringArray() { }

        
        public int Length()
        {
            return data.Length;
        }

        
        public void RemoveChar(char c)
        {
            data = data.Replace(c.ToString(), string.Empty);
        }

        
        public string GetData()
        {
            return data;
        }
    }
}

File: labsC++/lab02/StringArray.cpp
#include "StringArray.h"


StringArray::StringArray() : data("") {}


StringArray::StringArray(const std::string& str) : data(str) {}


StringArray::~StringArray() {}


size_t StringArray::length() const {
    return data.size();
}


void StringArray::removeChar(char c) {
    data.erase(remove(data.begin(), data.end(), c), data.end());
}


std::string StringArray::getData() const {
    return data;
}
